### DISCUSSION TITLE
Student application failing with no feedback

### DISCUSSION THREAD

**Discussion by J0J0HA** [community/community#56431] [GitHub Education]
### Hi everyone,

I have done 6 or 7 requests now, and issues are  getting repetitive.

The last three reviews, I just got this:

```
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on May 25, 2023 for @J0J0HA.

Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.


Once you have resolved the issues with your sumbission, visit us again and reapply.
```

Without any feedback. Before, i was able to work with the freedback messages, but now, it just says "did not work" without any explanation, and instantly (less then 30sec).

I have no idea what I am supposed to do now, because I used my school-email, which has the same domain as the schools website and my Student ID, back and front (I used an expored one for the back) (in german, with small english and france translations). I also have recorded a video to proof that the back is really the correct one, but noone will ever see it because I can only upload images.

Can anyone give me tips what I could add additionally to proof that I am a real student?

Only one more thing: I noticed something being off, I canged my profiles name to my real name, but in the report E-Mails i am always called by my old name. Is this just because of caching or did GitHub "not get" I canged my name.

**Comment by Iasimo92** [comment]
yes same here. Every time the same issue. Is there any other way to apply as a student and Github can verify it?

**Reply by J0J0HA** [reply]
No there isn't, unfortunately, GitHub has discontinued manual reviews. I have already tried to create a support ticket.

**Comment by samus-aran** [comment]
Hi @J0J0HA 👋 

When students receive this rejection reason (or lack of) we would usually suggest that you provide additional pieces of evidences. Ensure you select the most appropriate evidence type in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

Hope this helps. Let us know how you get on 👍
